wolfssl/wolfcrypt/sha3.h

```diff
@@ -78,7 +71,32 @@ typedef struct Sha3 {
 #endif /* WOLFSSL_ASYNC_CRYPT */
 } Sha3;
 
-#endif /* HAVE_FIPS */
+#ifdef HAVE_FIPS
+WOLFSSL_LOCAL int InitSha3_224_fips(Sha3* sha3);
```

Comment:
SHA-3 is a FIPS-140 algorithm that someone might want in the future.
No point merging this until someone wants it.